libs/SalesforceSDK/src/com/salesforce/androidsdk/rest/RestRequest.java

```diff
@@ -168,6 +172,7 @@ public String getPath(Object... args) {
 		}
 	}
 
+	private static final DateFormat ISO8601_DATE_FORMAT = new SimpleDateFormat ("yyyy-MM-dd'T'HH:mm:ss.SSSZ", Locale.US);
```

Comment:
Constant also define in MobileSync: https://github.com/forcedotcom/SalesforceMobileSDK-Android/blob/master/libs/MobileSync/src/com/salesforce/androidsdk/mobilesync/util/Constants.java#L82

Maybe we should have it in one place?

Reply:
We could make this `public` and reuse it in `MobileSync` since that library depends on this one anyway.

Reply:
Would we deprecate the MobileSync one? If we have to work around semver I was also thinking about making another constants class in SalesforceSDK with the date that the existing MobileSync class could extend

Reply:
Nah - that's too much work. Just update the constant in `MobileSync` to point to this one:
```
public static final DateFormat TIMESTAMP_FORMAT = RestRequest. ISO8601_DATE_FORMAT
```
That doesn't violate `semver`.

Reply:
Cool 👍